src/asset-picker/AssetPickerDropDown.js

```diff
@@ -35,26 +35,27 @@ export default class AssetPickerDropDown extends Component {
             router.push('asset-picker');
         }
     }
+    onClose(e) {
+        const { actions } = this.props;
+        e.stopPropagation();
+        actions.updateAssetPickerSearchQuery('');
+        actions.updateAssetPickerFilter('all');
```

Comment:
No, because first, it will explain the action much better.
Also, two actions cause two evaluations of all reducers, which is possibly a perf concern.